en/components/grids_templates/clipboard_interactions.md

```diff
@@ -0,0 +1,93 @@
+---
+}
+
+### @@igComponent Grid Clipboard Interactions
+Copy to clipboard operations are now available in the @@igComponent. This functionality provides a fast, easy and customizable way to copy data of the Angular Data Grid through the current multi cell data select. System Clipboard behavior gives the user ability to copy data from the @@igComponent into Excel or other external programs.
+
+#### Demo
+
+
+@@if (igxName === 'IgxGrid') {
+<div class="sample-container loading" style="height:635px">
+    <iframe id="grid-clipboard-operations-sample-iframe" src='{environment:demosBaseUrl}/grid/grid-clipboard-operations-sample' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="grid-clipboard-operations-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+}
+@@if (igxName === 'IgxTreeGrid') {
+<div class="sample-container loading" style="height:635px">
+    <iframe id="tree-grid-clipboard-operations-sample-iframe" src='{environment:demosBaseUrl}/tree-grid/tree-grid-clipboard-operations-sample' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="tree-grid-clipboard-operations-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+}
+
+### Functionality
+Copy behavior is working with the default interaction defined by the browser and operating system. Thus for the copy and paste behaviors, these are:
+
+- Windows/Unix based
+  - <kbd>Ctrl</kbd> + <kbd>C</kbd> / <kbd>Ctrl</kbd> + <kbd>Ins</kbd> as a keyboard shortcut
+  - <kbd>Ctrl</kbd> + <kbd>V</kbd> / <kbd>Shift</kbd> + <kbd>Ins</kbd> as a keyboard shortcut
+  - Copy action through the browser menu
+- macOS
+  - <kbd>⌘ Cmd</kbd> + <kbd>C</kbd> as a keyboard shortcut
+  - <kbd>⌘ Cmd</kbd> + <kbd>V</kbd> as a keyboard shortcut
+  - Copy action through the browser menu
+
+
+### Limitations
+- Both the **cut** and **copy** events are not natively supported in Internet Explorer. The exception is the
+**paste** event (IE 11) which is emitted but does not expose the `clipboardData` property in the event. 
+> [!NOTE] 
+> In order to `copy` cells in IE 11, you can use the keyboard selection. Hold the `shift key` in order to make a multi-cell selection, press `Ctrl + C` in order to copy.
+
+- The copy behavior is disabled while the grid is in edit mode.
+- The current version of this feature covers only the `copy` from grid behavior. Later on we plan to expose `paste` within grid behavior, for now you can use a custom paste handler for that purpose, have a look at our `Paste from Excel topic`.
```

Comment:
Only IgxGrid has Pase from Excel topic. An if statement should be added here. Also it will be nice if a link is added to the Paste from Excel Topic